SpatialGDK/Build/Programs/Improbable.Unreal.Scripts/WorkerCoordinator/LifetimeComponentTest.cs

```diff
@@ -0,0 +1,89 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Improbable.WorkerCoordinator
+{
+    /// <summary>
+    /// Lifetime component test code.
+    /// Call `LifetimeComponentTest.Run()` in `Program.Main` to start local test.
+    /// </summary>
+    internal class LifetimeComponentTest
```

Comment:
Just wanted to ask about the test. 

We determine whether the test has failed by looking at the logs and checking for errors correct?

Reply:
This test code provide a way that you can start debug on you local machine. You don't need to build coordinator and launch it to cloud deployment to see the result. 
With this code and output logs, you can identify if your logic is working well or not.
It will save a lot of time when you change some coordinator logic and want to check if it works well.